rites/roman1969/build/data-checks.ts

```diff
@@ -23,19 +23,19 @@
 import chalk from 'chalk';
 import { merge } from 'ts-deepmerge';
 
-import { Locale } from '../lib';
-import { Martyrology } from '../lib/catalog/martyrology';
-import { COLORS } from '../lib/constants/colors';
-import { PROPER_CYCLES, PSALTER_WEEKS, SUNDAY_CYCLES, WEEKDAY_CYCLES } from '../lib/constants/cycles';
-import { MONTHS } from '../lib/constants/months';
-import { RANKS } from '../lib/constants/ranks';
-import { SEASONS } from '../lib/constants/seasons';
-import { WEEKDAYS } from '../lib/constants/weekdays';
-import { GeneralRoman } from '../lib/general-calendar/proper-of-saints';
-import { locales } from '../lib/locales';
-import { CalendarDef } from '../lib/models/calendar-def';
-import { particularCalendars } from '../lib/particular-calendars';
-import { toPackageName } from '../lib/utils/string';
+import { Locale } from '../src';
+import { Martyrology } from '../src/catalog/martyrology';
+import { COLORS } from '../src/constants/colors';
+import { PROPER_CYCLES, PSALTER_WEEKS, SUNDAY_CYCLES, WEEKDAY_CYCLES } from '../src/constants/cycles';
+import { MONTHS } from '../src/constants/months';
+import { RANKS } from '../src/constants/ranks';
+import { SEASONS } from '../src/constants/seasons';
+import { WEEKDAYS } from '../src/constants/weekdays';
+import { GeneralRoman } from '../src/general-calendar/proper-of-saints';
+import { locales } from '../src/locales';
+import { CalendarDef } from '../src/models/calendar-def';
+import { particularCalendars } from '../src/particular-calendars';
+import { toPackageName } from '../src/utils/string';
```

Comment:
could we have internal aliases in each distinct package for it's src folder so we can simplify these, or for the distinct src directories so they're more easily referenced in a given package?
i.e. in order to produce

```suggestion
import { Locale } from '@src';
import { Martyrology } from '@src/catalog/martyrology';
import { COLORS } from '@src/constants/colors';
import { PROPER_CYCLES, PSALTER_WEEKS, SUNDAY_CYCLES, WEEKDAY_CYCLES } from '@src/constants/cycles';
import { MONTHS } from '@src/constants/months';
import { RANKS } from '@src/constants/ranks';
import { SEASONS } from '@src/constants/seasons';
import { WEEKDAYS } from '@src/constants/weekdays';
import { GeneralRoman } from '@src/general-calendar/proper-of-saints';
import { locales } from '@src/locales';
import { CalendarDef } from '@src/models/calendar-def';
import { particularCalendars } from '@src/particular-calendars';
import { toPackageName } from '@src/utils/string';
```

Reply:
I think it's a great idea. However, when defining an `@src` alias in the `tsconfig.json` of the current package, it completely overwrites the `paths` property from the parent `tsconfig.json` (instead of merging it). So, you would need to copy the `paths` config from the parent into the sub-package's `tsconfig.json` (in addition to the custom `@src`), which I find unfortunate.

I’m a bit torn between copying some config on all sub-packages to have cleaner paths in the code, or keeping the paths as they are now. But maybe I’m missing something, and there could be a better solution to handle these paths.

Reply:
ah, that's a fair problem... i wish we could use js configs instead of json configs :( let's keep it how you have for now, and we can iterate on it later.